docs/install.rst

```diff
@@ -1,55 +1,69 @@
 Install
 =======
 
-Below we assume you have the default Python environment already configured on
+Below we assume you have the default Python3 environment already configured on
 your computer and you intend to install ``mvlearn`` inside of it.  If you want
 to create and work with Python virtual environments, please follow instructions
 on `venv <https://docs.python.org/3/library/venv.html>`_ and `virtual
 environments <http://docs.python-guide.org/en/latest/dev/virtualenvs/>`_.
 
-First, make sure you have the latest version of ``pip`` (the Python package manager)
+First, make sure you have the latest version of ``pip3`` (the Python3 package manager)
 installed. If you do not, refer to the `Pip documentation
-<https://pip.pypa.io/en/stable/installing/>`_ and install ``pip`` first.
+<https://pip.pypa.io/en/stable/installing/>`_ and install ``pip3`` first.
 
 Install the released version
 ----------------------------
 
-Install the current release of ``mvlearn`` with ``pip``::
+Install the current release of ``mvlearn`` with ``pip3``::
 
-    $ pip install mvlearn
+    $ pip3 install mvlearn
 
 To upgrade to a newer release use the ``--upgrade`` flag::
 
-    $ pip install --upgrade mvlearn
+    $ pip3 install --upgrade mvlearn
 
```

Comment:
How do you upgrade if you want the torch dependencies? Does this single command do it, or do you need something like 

pip install --upgrade mvlearn[torch]

Reply:
see below. Tell me if refined documentation is needed.